apps/ledger-live-mobile/src/screens/BuyDeviceScreen.tsx

```diff
@@ -98,6 +124,13 @@ export default function BuyDeviceScreen() {
 
   return (
     <StyledSafeAreaView>
+      {discreetMode && (
```

Comment:
```suggestion
      {discreetMode ? (<Component />) : null}
```
cf : https://ledger.slack.com/archives/C0308JT8WS2/p1651831403935859